index.html

```diff
@@ -248,11 +249,11 @@ <h3>
         <p>
           The `ongeometrychange` attribute is an event handler whose
           corresponding event handler event type is "change". The event is
-          dispatched when the available title bar region changes, e.g., in
+          dispatched when there the [=title bar region resizes=], e.g., in
```

Comment:
typo "when *there* the title bar region"

Reply:
fixed typo